boards/arm/bl5340_dvk/bl5340_dvk_cpuapp_common.dts

```diff
@@ -195,14 +196,16 @@
 	mosi-pin = <9>;
 	sck-pin = <8>;
 
-	ili9340@0 {
+	ili9340: ili9340@0 {
 		compatible = "ilitek,ili9340";
 		reg = <0>;
 		spi-max-frequency = <32000000>;
 		label = "DISPLAY";
 		reset-gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
 		cmd-data-gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
 		rotation = <270>;
+		width = <240>;
+		height = <320>;
```

Comment:
I think these sorts of changes should be put into the previous patch which added the properties to the binding, for bisectability.